src/codesyncd/handlers/tab_handler.ts

```diff
@@ -37,6 +32,7 @@ export class TabHandler {
 		const is_relative = isRelativePath(relative);
         if (!(is_relative && fs.existsSync(tabFilePath))) return;
 		removeTabFile(tabFilePath, "removeTabFile");
+		CodeSyncLogger.debug(`TAB REMOVED: ${tabFilePath}`);
```

Comment:
I think this was for debugging

Reply:
Yes but I was thinking if we can keep this? as it can give us insights when a tab file is removed. What do you think?